Directory.Build.props

```diff
@@ -7,10 +7,11 @@
     <CheckEolTargetFramework>false</CheckEolTargetFramework>
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
+    <Copyright>Copyright © 2013-23 Serilog Contributors</Copyright>
```

Comment:
This flows up to the nuget landing page/ package metadata (and I think the FILEVERSIONINFO on the DLL)

Reply:
Hm the Authors in the package (i.e., `<Authors>Serilog Contributors</Authors>`) might get appended in some renderings?

Reply:
Seems they are complementary per https://learn.microsoft.com/en-us/nuget/create-packages/package-authoring-best-practices#copyright, so leaving it that way